modules/troubleshooting-dspa-component-errors.adoc

```diff
@@ -0,0 +1,221 @@
+endif::[]
+
+ifdef::disconnected[]
+For more information, see link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}_in_a_disconnected_environment/working-with-certificates_certs[Working with certificates].
+endif::[]
+endif::[]
+
+|`ObjectStorageAvailable`
+
+Ready
+|False
+
+False
+|`Could not connect to Object Store
+Deployment for component "ds-pipeline-pipelines-definition" is missing - pre-requisite component may not yet be available.
+Deployment for component "ds-pipeline-persistenceagent-pipelines-definition" is missing - pre-requisite component may not yet be available.
+Deployment for component "ds-pipeline-scheduledworkflow-pipelines-definition" is missing - pre-requisite component may not yet be available.`
+|In clusters running RHOAI 2.8.x, the Data Science Pipelines Manager may fail to connect to the object storage, and the Pipeline Server may not be created. This can happen for several reasons.
+
+Ensure that your object store credentials and connection information are accurate, and verify that the object store is accessible from within the DS Project's associated OpenShift Namespace. One common issue is that the object storage SSL certificate may not be trusted, particularly if self-signed certificates are used.
+
+Verify and update your object storage credentials, then retry the operation.
+
+|`ObjectStorageAvailable`
+
+Ready
+|False
+
+False
+|`Wrong credentials for Object Storage: Could not connect to (minio-my-project.apps.my-cluster.com), Error: The request signature we calculated does not match the signature you provided. Check your key and signing method.`
+|Provide the correct credentials for your object storage and retry the operation.
+
+|`ObjectStorageAvailable`
+
+Ready
+|False
+
+False
+|`FailingToDeploy: Couldn’t connect to (minio-myproject.namespace.svc.cluster.local.9000), Error: Get “http://minio-myproject.namespace.svc.cluster.local.9000/mlpipeline/?location=”`
+|
+
+This is an internal API used for development purposes. If the error persists beyond 60 seconds, investigate for underlying issues related to misconfigurations or network problems.
+
+
+|`DatabaseAvailable`
+
+Ready
+|False
+
+False
+|`FailingToDeploy: Dial tcp XXX.XX.XXX.XXX:3306 : i/o timeout`
+|If the issue persists beyond startup, check for network issues or misconfigurations in the database connection settings.
+
+|`DatabaseAvailable`
+
+Ready
+|False
+
+False
+|`Unable to connect to external database: tls: failed to verify certificate: x509: certificate signed by unknown authority`
+|This issue can occur when you use an external database, such as Amazon RDS. The data science pipelines manager is unable to connect to the database as it does not trust the database SSL certificate. This prevents the creation of the pipeline server. This issue can occur with any external database solution. Contact your IT operations administrator to add the relevant certificates. 
```

Comment:
Suggestion:
`This issue can occur when you use any external database, such as Amazon RDS. The data science pipelines manager cannot connect to the database because it does not trust the database SSL certificate, preventing the creation of the pipeline server.  Contact your IT operations administrator to add the relevant certificates.`

Reply:
Changed to your suggestion.